api/src/test/java/org/jboss/marshalling/cloner/SerializingClonerTestCase.java

```diff
@@ -237,6 +241,40 @@ public void testObjectWithSelectiveSerialization() throws Throwable {
JAAgAKgwAKwAsAQAKYm9vbGVhblZhbAEAAVoSAAAALgwALwAwAQAIdG9TdHJpbmcBADsoTG9yZy9qYm9zcy9tYXJzaGFsbGluZy9jbG9uZXIvTXlSZWNvcmQ7KUxqYXZhL2xhbmcvU3RyaW5nOxIAAAAyDAAzADQBAAhoYXNoQ29kZQEAKihMb3JnL2pib3NzL21hcnNoYWxsaW5nL2Nsb25lci9NeVJlY29yZDspSRIAAAA2DAA3ADgBAAZlcXVhbHMBADwoTG9yZy9qYm9zcy9tYXJzaGFsbGluZy9jbG9uZXIvTXlSZWNvcmQ7TGphdmEvbGFuZy9PYmplY3Q7KVoBAB0oTGphdmEvbGFuZy9TdHJpbmc7Q0pJU0JGRFopVgEABENvZGUBAA9MaW5lTnVtYmVyVGFibGUBABJMb2NhbFZhcmlhYmxlVGFibGUBAAR0aGlzAQAnTG9yZy9qYm9zcy9tYXJzaGFsbGluZy9jbG9uZXIvTXlSZWNvcmQ7AQAQTWV0aG9kUGFyYW1ldGVycwEAFCgpTGphdmEvbGFuZy9TdHJpbmc7AQADKClJAQAVKExqYXZhL2xhbmcvT2JqZWN0OylaAQABbwEAEkxqYXZhL2xhbmcvT2JqZWN0OwEAAygpQwEAAygpSgEAAygpUwEAAygpQgEAAygpRgEAAygpRAEAAygpWgEAClNvdXJjZUZpbGUBAA1NeVJlY29yZC5qYXZhAQAGUmVjb3JkAQAQQm9vdHN0cmFwTWV0aG9kcw8GAFEKAFIAUwcAVAwAVQBWAQAfamF2YS9sYW5nL3J1bnRpbWUvT2JqZWN0TWV0aG9kcwEACWJvb3RzdHJhcAEAsShMamF2YS9sYW5nL2ludm9rZS9NZXRob2RIYW5kbGVzJExvb2t1cDtMamF2YS9sYW5nL1N0cmluZztMamF2YS9sYW5nL2ludm9rZS9UeXBlRGVzY3JpcHRvcjtMamF2YS9sYW5nL0NsYXNzO0xqYXZhL2xhbmcvU3RyaW5nO1tMamF2YS9sYW5nL2ludm9rZS9NZXRob2RIYW5kbGU7KUxqYXZhL2xhbmcvT2JqZWN0OwgAWAEAT3N0cmluZ1ZhbDtjaGFyVmFsO2xvbmdWYWw7aW50VmFsO3Nob3J0VmFsO2J5dGVWYWw7ZmxvYXRWYWw7ZG91YmxlVmFsO2Jvb2xlYW5WYWwPAQAHDwEADQ8BABEPAQAVDwEAGQ8BAB0PAQAhDwEAJQ8BACkBAAxJbm5lckNsYXNzZXMHAGQBACVqYXZhL2xhbmcvaW52b2tlL01ldGhvZEhhbmRsZXMkTG9va3VwBwBmAQAeamF2YS9sYW5nL2ludm9rZS9NZXRob2RIYW5kbGVzAQAGTG9va3VwADEACAACAAAACQASAAsADAAAABIADwAQAAAAEgATABQAAAASABcAGAAAABIAGwAcAAAAEgAfACAAAAASACMAJAAAABIAJwAoAAAAEgArACwAAAANAAEABQA5AAIAOgAAALwAAwAMAAAAOCq3AAEqK7UAByoctQANKiG1ABEqFQW1ABUqFQa1ABkqFQe1AB0qFwi1ACEqGAm1ACUqFQu1ACmxAAAAAgA7AAAABgABAAAAAwA8AAAAZgAKAAAAOAA9AD4AAAAAADgACwAMAAEAAAA4AA8AEAACAAAAOAATABQAAwAAADgAFwAYAAUAAAA4ABsAHAAGAAAAOAAfACAABwAAADgAIwAkAAgAAAA4ACcAKAAJAAAAOAArACwACwA/AAAAJQkACwAAAA8AAAATAAAAFwAAABsAAAAfAAAAIwAAACcAAAArAAAAEQAvAEAAAQA6AAAAMQABAAEAAAAHKroALQAAsAAAAAIAOwAAAAYAAQAAAAMAPAAAAAwAAQAAAAcAPQA+AAAAEQAzAEEAAQA6AAAAMQABAAEAAAAHKroAMQAArAAAAAIAOwAAAAYAAQAAAAMAPAAAAAwAAQAAAAcAPQA+AAAAEQA3AEIAAQA6AAAAPAACAAIAAAAIKiu6ADUAAKwAAAACADsAAAAGAAEAAAADADwAAAAWAAIAAAAIAD0APgAAAAAACABDAEQAAQABAAsAQAABADoAAAAvAAEAAQAAAAUqtAAHsAAAAAIAOwAAAAYAAQAAAAMAPAAAAAwAAQAAAAUAPQA+AAAAAQAPAEUAAQA6AAAALwABAAEAAAAFKrQADawAAAACADsAAAAGAAEAAAADADwAAAAMAAEAAAAFAD0APgAAAAEAEwBGAAEAOgAAAC8AAgABAAAABSq0ABGtAAAAAgA7AAAABgABAAAAAwA8AAAADAABAAAABQA9AD4AAAABABcAQQABADoAAAAvAAEAAQAAAAUqtAAVrAAAAAIAOwAAAAYAAQAAAAMAPAAAAAwAAQAAAAUAPQA+AAAAAQAbAEcAAQA6AAAALwABAAEAAAAFKrQAGawAAAACADsAAAAGAAEAAAADADwAAAAMAAEAAAAFAD0APgAAAAEAHwBIAAEAOgAAAC8AAQABAAAABSq0AB2sAAAAAgA7AAAABgABAAAAAwA8AAAADAABAAAABQA9AD4AAAABACMASQABADoAAAAvAAEAAQAAAAUqtAAhrgAAAAIAOwAAAAYAAQAAAAMAPAAAAAwAAQAAAAUAPQA+AAAAAQAnAEoAAQA6AAAALwACAAEAAAAFKrQAJa8AAAACADsAAAAGAAEAAAADADwAAAAMAAEAAAAFAD0APgAAAAEAKwBLAAEAOgAAAC8AAQABAAAABSq0ACmsAAAAAgA7AAAABgABAAAAAwA8AAAADAABAAAABQA9AD4AAAAEAEwAAAACAE0ATgAAADgACQALAAwAAAAPABAAAAATABQAAAAXABgAAAAbABwAAAAfACAAAAAjACQAAAAnACgAAAArACwAAABPAAAAHAABAFAACwAIAFcAWQBaAFsAXABdAF4AXwBgAGEAYgAAAAoAAQBjAGUAZwAZ");
```

Comment:
This is a little crazy but it was the most straightforward way to test this without introducing CI problems due to class file version errors.